From 05e69f495f2a23b294601720962775c0cf617e76 Mon Sep 17 00:00:00 2001
From: Hans Rosenfeld <hans.rosenfeld@joyent.com>
Date: Wed, 31 Jan 2018 13:19:00 +0100
Subject: [PATCH] OS-6498 bnxe does not attach on debug kernels Reviewed by:
 Patrick Mooney <patrick.mooney@joyent.com> Reviewed by: Mike Gerdts
 <mike.gerdts@joyent.com>

---
 usr/src/uts/common/io/bnxe/bnxe_debug.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/usr/src/uts/common/io/bnxe/bnxe_debug.c b/usr/src/uts/common/io/bnxe/bnxe_debug.c
index 1411045f2c..e8587a17b3 100644
--- a/usr/src/uts/common/io/bnxe/bnxe_debug.c
+++ b/usr/src/uts/common/io/bnxe/bnxe_debug.c
@@ -186,7 +186,7 @@ void BnxeLogWarn(void * pDev,
 }
 
 
-#ifdef DBG
+#ifdef DEBUG
 
 void BnxeLogDbg(void * pDev,
                 char * pFmt,
@@ -209,7 +209,7 @@ void BnxeLogDbg(void * pDev,
     cmn_err(CE_WARN, "!%s: %s", BnxeDevName(pUM), buf);
 }
 
-#endif /* DBG */
+#endif /* DEBUG */
 
 
 void BnxeDumpMem(um_device_t * pUM,
-- 
2.21.0

